extension-iap/src/iap_ios.mm

```diff
@@ -506,6 +504,7 @@ static int IAP_GetProviderId(lua_State* L)
     {"restore", IAP_Restore},
     {"set_listener", IAP_SetListener},
     {"get_provider_id", IAP_GetProviderId},
+    {"process_pending_transactions", IAP_ProcessPendingTransactions},
```

Comment:
Seeing as you've added this function, I assume you intended a major version change, but forgot to flag for it in the description?

Reply:
Could you please add documentation for the new method as well?

Reply:
Agreed, please add documentaiton.

Reply:
@dyukorev Could you please address this?